components/clp-package-utils/clp_package_utils/scripts/search.py

```diff
@@ -76,46 +79,28 @@ def main(argv):
         clp_config.validate_logs_dir()
 
         # Validate and load necessary credentials
-        validate_and_load_db_credentials_file(clp_config, clp_home, True)
+        validate_and_load_db_credentials_file(clp_config, clp_home, False)
```

Comment:
Looks like a bug to me. Technically we can also factor out this `validate and load config block` into a common function and the submitter scripts (not the ones in native) can share it.  Just need to find a different name.

Reply:
Makes sense